Scripts/Restriction/ranacompiler.py

```diff
@@ -977,10 +977,10 @@ def information_mixer(self, file1, file2, file3):
                 except OverhangError:  # overhang error
                     n = name  # do not include the enzyme
                     if not bl[2]:
-                        print("Anyway, %s is not commercially available.\n" % n)
+                        print(f"Anyway, {n} is not commercially available.\n")
                     else:
                         print(
-                            "Unfortunately, %s is commercially " % n + " available.\n"
```

Comment:
I can see why after flynt we get a flake8 error - might be worth fixing the original now anyway on a tiny pull request?

Reply:
Fixed in #2572